OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/Analytics.java

```diff
@@ -493,6 +493,37 @@ void trackValuePropLearnMoreTapped(@NonNull String courseId, @Nullable String as
      */
     void trackLockedContentTapped(@NonNull String courseId, @NonNull String assignmentId);
 
+    /**
+     * Track the tapped occurrence when a user taps "Upgrade Now" in an embedded or modal value prop.
+     *
+     * @param courseId     course id of the course on which button is tapped
+     * @param price        Price for which the course can upgrade
+     * @param sectionId    Section id of course unit
+     * @param subSectionId SubSection id of course unit
+     * @param unitId       Unit id of course unit
+     * @param componentId  Component id of course unit
+     * @param isSelfPaced  Whether the course is self-paced or instructor paced
+     */
+    void trackUpgradeNowTapped(@NonNull String courseId, @NonNull String price, @Nullable String sectionId,
+                               @Nullable String subSectionId, @Nullable String unitId,
+                               @Nullable String componentId, @NonNull boolean isSelfPaced);
+
+    /**
+     * Track the tapped occurrence when a user taps "Show more/Show less" in an embedded value prop.
+     *
+     * @param courseId     course id of the course on which button is tapped
+     * @param price        Price for which the course can upgrade
+     * @param sectionId    Section id of course unit
+     * @param subSectionId SubSection id of course unit
+     * @param unitId       Unit id of course unit
+     * @param componentId  Component id of course unit
+     * @param isSelfPaced  Whether the course is self-paced or instructor paced
+     * @param showMore     True when user tapped on show more otherwise false
+     */
+    void trackValuePropShowMoreLessTapped(@NonNull String courseId, @NonNull String price, @Nullable String sectionId,
```

Comment:
rename the method to `trackValuePropShowMoreLessClicked` and `sectionId`, `subSectionId`, `unitId` params. Also update the method calls accordingly.